src/pip/_internal/legacy_resolve.py

```diff
@@ -362,7 +360,9 @@ def _resolve_one(
         # register tmp src for cleanup in case something goes wrong
         requirement_set.reqs_to_cleanup.append(req_to_install)
 
-        abstract_dist = self._get_abstract_dist_for(req_to_install)
+        abstract_dist = self._get_abstract_dist_for(
+            req_to_install, require_hashes
```

Comment:
nit: trailing comma

Reply:
Since the Black PR and general consensus to move in that direction I have been omitting these to reduce churn when we do eventually get that enabled.